### DISCUSSION TITLE
soil humidity sensor

### DISCUSSION THREAD

**Discussion by ricardogusi** [Pi4J/pi4j#135] [Q&A] [answered]
Hi, can I use pi4j for soil humidity sensor ? didn't find anything about it

**Comment by FDelporte** [comment, marked as answer]
Which signal would be provided by the sensor?
I found this one: https://www.sparkfun.com/products/13322 and according to this tutorial https://www.instructables.com/Soil-Moisture-Sensor-Raspberry-Pi/ the signal is handled as on/off digital input. In that case, Pi4J can definitely be used for this use case.

**Reply by ricardogusi** [reply]
Wow, good to know! Thank You
